JS101/fundamentals/rock-paper-scissors-project.md

```diff
@@ -93,6 +93,11 @@ Submit a solution with a pull request to this [file](https://github.com/TheOdinP
 - [Cody Selman's solution](https://github.com/CodySelman/rock-paper-scissors) - [View in browser](https://codyselman.github.io/rock-paper-scissors/index.html)
 - [Zach Coursey's solution](https://github.com/zcoursey22/rock-paper-scissors) - [View in browser](https://zcoursey22.github.io/rock-paper-scissors)
 - [Nikhil Patel's Solution](https://github.com/nikhilpatel87/rock-paper-scissors/blob/master/index.html)
+<<<<<<< HEAD
```

Comment:
Hi @Jacob-Folley 

There was a difference in the file you submitted and the one stored on the master branch here which has resulted in a git diff which needs to be resolved. You need to pull down the master branch into your forked copy of the javascript repo. If you aren't sure how to do that then you can delete the fork and then create a new one and submit your PR then.